tests/unit/test_routes.py

```diff
@@ -763,10 +718,11 @@ def test_validate_resource(module_client, module_db, fake_auth_from_oc):
                           )
 
     assert (response.status_code == 422)
-    assert (isinstance(response.json.get('errors').get(INVALID_PARAMS), dict))
-    assert (isinstance(response.json.get('errors').get(INVALID_PARAMS).get('message'), str))
-    assert ("url" in response.json.get('errors').get(INVALID_PARAMS).get("params"))
```

Comment:
We also need to put the decoupled responses back in here, they were lost during the rebase